src/core/src/OmniTileset.cpp

```diff
@@ -509,6 +528,57 @@ void OmniTileset::addImageryIon(const pxr::SdfPath& imageryPath) {
     _imageryPaths.push_back(imageryPath);
 }
 
+void OmniTileset::addImageryPolygon(const pxr::SdfPath& imageryPath) {
+    const OmniPolygonImagery imagery(imageryPath);
+
+    const auto uniqueName = "imagery_polygon_test"; // DEVEL
```

Comment:
Would `imagery.getName()` work here?

Reply:
Yes. That's better. Looks like `uniqueName` in `getImagery` (now `getImageryIon`) was removed when I last merged to `main`.

With other imagery/raster-overlay types coming, it seems like having an overladed getIimagery/getRasterOverlay function would prevent code splintering from copied logic. I'll try to consolidate these methods when adding the next raster-overlay type.